README.md

```diff
@@ -108,8 +108,8 @@ extern "C" void GDN_EXPORT godot_nativescript_init(void *handle)
 # Compiling
 ```
 $ cd SimpleLibrary
-$ clang -fPIC -o src/init.os -c src/init.cpp -g -O3 -std=c++14 -I../godot-cpp/include -Igodot_headers
-$ clang -o lib/libtest.so -shared src/init.os -L../godot-cpp/lib -lgodot-cpp
+$ clang -fPIC -o src/init.os -c src/init.cpp -g -O3 -std=c++14 -I../godot-cpp/include -I../godot_headers
+$ clang -o lib/libtest.so -shared src/init.os -L../godot-cpp/lib -Lgodot-cpp
```

Comment:
Hmmm, lowercase `l` is for the library itself, uppercase `L` is for specifying a directory to search in, so in this case the declaration of the library to be used is not there anymore?